File: IFRS - Prog II/250730-IFRS-ADS-Prog2/aula05/src/tarefas/Exercice1/Computador.java
package Exercice1;

public class Computador {

    public String marca;
    public String modelo;
    public int ano;

    public void print() {
        System.out.println("Computador");
        System.out.println("  ├─ Marca: " + marca);
        System.out.println("  ├─ Modelo: " + modelo);
        System.out.println("  └─ Ano: " + ano);
    }
}

File: IFRS - Prog III/aula-08/Exercicio01/Computador.js
class Computador {

    constructor(marca, modelo, processador, memoriaRAM, armazenamento) {
        this.marca = marca;
        this.modelo = modelo;
        this.processador = processador;
        this.memoriaRAM = memoriaRAM;
        this.armazenamento = armazenamento;
    }

    printInfo(){
        console.log(`Marca: ${this.marca}`);
        console.log(`Modelo: ${this.modelo}`);
        console.log(`Processador: ${this.processador}`);
        console.log(`Memória RAM: ${this.memoriaRAM} GB`);
        console.log(`Armazenamento: ${this.armazenamento} GB`);
    }
}

module.exports = Computador;